res/css/views/settings/tabs/user/_LabsUserSettingsTab.scss

```diff
@@ -15,18 +15,8 @@ limitations under the License.
 */
 
 .mx_LabsUserSettingsTab {
-    .mx_SettingsTab_subsectionText, .mx_SettingsTab_section {
-        margin-bottom: 30px;
-    }
-
-    .mx_SettingsTab_section .mx_SettingsFlag {
+    .mx_SettingsFlag {
```

Comment:
Lab sections are now having a 24px bottom margin:

![image](https://user-images.githubusercontent.com/6216686/173291871-01509125-c81b-4df4-bf32-76af0e338564.png)

Preferences have 30. We should keep that in sync.

Reply:
@weeman1337 Yeah I noticed that too, and I think we should create a class like `mx_SettingsTab_section--wideMargin` (or something like that) instead of setting a value for each case, in order to normalize the wide spacing for other tabs too. How does that sound?

Reply:
„wide“ sounds like horizontal margin. I mentioned the bottom margin…

Reply:
yes my bad, let me try to find a good word for that...

Reply:
My point here is that your changes remove the `30px` bottom margin we had before. `30px` is the same as in preferences. We should keep that in sync and should not use different margins in each view.

Reply:
Yes, I have noticed that. So does creating a variable to ensure the same margin value is used both on labs tab and preferences tab sound good? :thinking:  IMO reducing the number of magic numbers should contribute to improve the quality of the code.